2-Authorization-I/1-call-graph/SPA/src/utils/claimUtils.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * Populate claims table with appropriate description
+ * @param {Object} claims ID token claims
+ * @returns claimsObject
+ */
+export const createClaimsTable = (claims) => {
+    let claimsObj = {};
+    let index = 0;
+
+    Object.keys(claims).map((key) => {
+        switch (key) {
+            case 'aud':
+                populateClaim(
+                    key,
+                    claims[key],
+                    "Identifies the intended recipient of the token. In ID tokens, the audience is your app's Application ID, assigned to your app in the Azure portal.",
+                    index,
+                    claimsObj
+                );
+                index++;
+                break;
+            case 'iss':
+                populateClaim(
+                    key,
+                    claims[key],
+                    'Identifies the issuer, or authorization server that constructs and returns the token. It also identifies the Azure AD tenant for which the user was authenticated. If the token was issued by the v2.0 endpoint, the URI will end in /v2.0. The GUID that indicates that the user is a consumer user from a Microsoft account is 9188040d-6c67-4c5b-b112-36a304b66dad.',
+                    index,
+                    claimsObj
+                );
+                index++;
+                break;
+            case 'iat':
+                populateClaim(
+                    key,
+                    changeDateFormat(claims[key]),
+                    'Issued At indicates when the authentication for this token occurred.',
```

Comment:
>Issued At indicates when the authentication for this token occurred. [](http://example.com/codeflow?start=21&length=68)

"Issued At" indicates the timestamp when the authentication for this user occurred.

Reply:
Updated